src/login/login_auth.cpp

```diff
@@ -248,7 +253,7 @@ int32 login_parse(int32 fd)
                 fmtQuery = "INSERT INTO accounts(id,login,password,timecreate,timelastmodify,status,priv)\
 									   VALUES(%d,'%s',PASSWORD('%s'),'%s',NULL,%d,%d);";
 
-                if (Sql_Query(SqlHandle, fmtQuery, accid, name.c_str(), password.c_str(),
+                if (Sql_Query(SqlHandle, fmtQuery, accid, escaped_name, escaped_pass,
```

Comment:
isn't escaped_pass empty here? it was only escaped inside LOGIN_CREATE

Reply:
fixed. dunno how I lost the line. was pushing from the build I was using.